app/controllers/placements/schools/placements_controller.rb

```diff
@@ -33,6 +36,10 @@ def set_decorated_placement
     @placement = @placement.decorate
   end
 
+  def set_decorated_school
+    @school = @school.decorate
+  end
```

Comment:
I think you should be able to use `@placement.school` in your view and already have a decorated school object in your hand.

This is because:

https://github.com/DFE-Digital/itt-mentor-services/blob/a82c7b46f7f14d496184b3192ca5acc3cee56b46/app/decorators/placement_decorator.rb#L1-L3